demos/python_demos/common/models/retinaface.py

```diff
@@ -75,6 +80,8 @@ def __init__(self, detect_attributes=False):
             self._features_stride_fpn, [anchors.shape[0] for anchors in self._anchors_fpn.values()]
         ))
         self.landmark_std = 0.2 if detect_attributes else 1.0
+        self.nms_threshold = 0.5 if process_landmarks else 0.3
+        self.include_boundaries = False if process_landmarks else True
```

Comment:
```suggestion
        self.include_boundaries = not process_landmarks
```

Reply:
Done

Reply:
Come to think of it, if `include_boundaries` is just `not process_landmarks`, is it even necessary to store it in the class? Seems like it's only used in the `nms` call, which you could just edit to `nms(..., include_boundaries=not process_landmarks)`.

Reply:
`include_boundaries` variable has been removed, ` not process_landmarks` is used instead